### PR TITLE
Highlight search text in SearchActivity

### LINKED ISSUE #599: Highlight search text in SearchActivity
## Kind (Required)

- Improvement

 ## Overview (Required)

- I think it is useful 🙏 

 ## Links	

-

### PR DESCRIPTION
**Pull request by t-nonomura**
## Issue
- close #599 

## Overview (Required)
- Searched words are highlighted on the search result screen, session details screen and speaker details screen.

## Links
-

## Screenshot
Before | After
:--: | :--:
 | 
 | 
 |

### CODE REVIEW

File: feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetSessionsFragment.kt [hunk 1]
```diff
@@ -138,7 +138,7 @@ class BottomSheetSessionsFragment : Fragment(R.layout.fragment_bottom_sheet_sess
             binding.isFiltered = uiModel.filters.isFiltered()
             binding.filteredSessionCount.isVisible = uiModel.filters.isFiltered()
             groupAdapter.update(sessions.map {
-                sessionItemFactory.create(it, sessionsViewModel)
```
Comment (takahirom) [inline feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetSessionsFragment.kt]: Can we use default arguments? (Will it error?)

**Comment by takahirom** [comment]
Sorry. Please fix conflict 🙇

File: feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetSessionsFragment.kt (hunk 1 above)
Comment (t-nonomura) [reply feature/session/src/main/java/io/github/droidkaigi/confsched2020/session/ui/BottomSheetSessionsFragment.kt]: That's right, fixed it !
And also set default arguments ​​for other Factories that use the same parameters.

**Comment by t-nonomura** [comment]
Conflicts resolved 🙏

**Pull request merged**